cassava_trait.obo

```diff
@@ -9051,6 +9052,111 @@ relationship: variable_of CO_334:0100692 ! um
+relationship: variable_of CO_334:0002094 ! Breakdown
+relationship: variable_of CO_334:0010536 ! Calculation: Breakdown_method
+relationship: variable_of CO_334:0100704 ! pascal per seconds
+created_by: Ukoabasi Ekanem
+creation_date: 10/2/2023
+
+[Term]
+id: CO_334:0004041
+name: Final viscosity of gari estimation in pascal per seconds
+namespace: cassava_trait
+def: "Viscosity at the end of the RVA test. Measured using the Rapid Visco Analyzer (RVA) in pascal per second." []
+synonym: "RVA-final visc (RUV)" EXACT []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0002092 ! Final viscosity
+relationship: variable_of CO_334:0010535 ! Estimation: Final viscosity_method
+relationship: variable_of CO_334:0100704 ! pascal per seconds
+created_by: Ukoabasi Ekanem
+creation_date: 10/2/2023
+
+[Term]
+id: CO_334:0004042
+name: Pasting temperature of gari measurement in degree celsius
+namespace: cassava_trait
+def: "Temperature at which the viscosity trace first raises above the baseline during the initial heating phase measured by the Rapid Visco Analyser (RVA) in degree celsius." []
+synonym: "RVA-pasting temp (°C)" EXACT []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0002100 ! Pasting temperature
+relationship: variable_of CO_334:0010539 ! Measurement: Pasting temperature_method
+relationship: variable_of CO_334:0100703 ! degree celsius
+created_by: Ukoabasi Ekanem
+creation_date: 10/2/2023
+
+[Term]
+id: CO_334:0004043
+name: Peak viscosity of gari measurement in degree celsius
+namespace: cassava_trait
+def: "Maximum viscosity during the heating or hold phase of the RVA test measured by the Rapid Visco Analyser (RVA) in degree celsius." []
+synonym: "RVA-peak visc(RUV)" EXACT []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0002098 ! Peak viscosity
+relationship: variable_of CO_334:0010538 ! Measurement: Peak viscosity_method
+relationship: variable_of CO_334:0100704 ! pascal per seconds
+created_by: Ukoabasi Ekanem
+creation_date: 10/2/2023
+
+[Term]
+id: CO_334:0004044
+name: Peak time viscosity of gari measured in minutes
+namespace: cassava_trait
+def: "Time at which the viscosity of gari sample reaches its maximum value during the RVA test. Measured using the Rapid Visco Analyzer (RVA) in pascal per second." []
+synonym: "RVA-peak time (m)" EXACT []
+is_a: CO_334:0001000 ! Variables
+relationship: variable_of CO_334:0004045 ! Measurement: Peak time viscosity_method
+relationship: variable_of CO_334:0004046 ! Peak time viscosity
+relationship: variable_of CO_334:0100696 ! minute
+created_by: Ukoabasi Ekanem
+creation_date: 10/2/2023
+
+[Term]
+id: CO_334:0004045
+name: Measurement: Peak time viscosity_method
+namespace: CassavaMethod
+def: "Maximum time taken for the viscosity to reach its peak value during the RVA test" []
+is_a: CO_334:0001002 ! Methods
+relationship: method_of CO_334:0002098 ! Peak viscosity
+
+[Term]
+id: CO_334:0004046
+name: Peak time viscosity
```

Comment:
is this different than the trait CO_334:0002098 "peak viscosity" ?

Reply:
> is this different than the trait CO_334:0002098 "peak viscosity" ?

Yes